README.md

```diff
@@ -1,24 +1,299 @@
-# README
+<!-- TABLE OF CONTENTS -->
```

Comment:
Please check that there is an author acknowledgment:
![image](https://user-images.githubusercontent.com/90795408/215203357-46867720-039c-4ed7-a63b-908106ebc98c.png)

Reply:
> Please check that there is an author acknowledgment: ![image](https://user-images.githubusercontent.com/90795408/215203357-46867720-039c-4ed7-a63b-908106ebc98c.png)

@robinbenitezmora thanks for pointing this out.  That review comment was only regarding the frontend repo, because, at the time of the review, I couldn't find the Microverse required readme file.  I included the comment here because there was no open for me to review PR in that project.

If you need guidance/assistance, please don't hesitate to contact me.